### DISCUSSION TITLE
Installer doesn’t finish

### DISCUSSION THREAD

**Discussion by franklw262** [flybywiresim/aircraft#4330] [Help]
When I run the new installer with the stable 5.4 version the installer is stuck on finishing update

**Comment by ExampleWasTaken** [comment]
Try to empty your community folder and then try again. Also, try to download other versions and see if the issue persists.

**Comment by franklw262** [comment]
Community folder totally empty
When I use version 1 of installer it automatically updates to latest version

**Reply by ExampleWasTaken** [reply]
What does "version 1 of the installer" mean? Please make sure you are on the latest version of the installer. The installer can be downloaded here: https://api.flybywiresim.com/installer
The current version of the installer is `1.1.4` you can check the version of your current install by clicking on the "Settings" button in the lower left corner of the installer.

**Comment by franklw262** [comment]
Yes I have been using 1.1.4 and 5.4 but it locks up during finishing update

**Reply by ExampleWasTaken** [reply]
I'm out of ideas at this point, but maybe @nistei can help you further. Possibly by taking a look at your logs.

**Reply by nistei** [reply]
@franklw262 Can you try this please:
- Press `Ctrl+F12` to open the developer tools.
- Switch to the `Console` tab in the window that opened.
- Start an install
- Wait till it is stuck
- Click on any line in the `Console` tab then right-click it
- Select `Save As` and send me the file here or on Discord to nistei#1362

This way I can analyze and check what is happening.

**Reply by ExampleWasTaken** [reply]
> 
> 
> @franklw262 Can you try this please:
> 
>     * Press `Ctrl+F12` to open the developer tools.
> 
>     * Switch to the `Console` tab in the window that opened.
> 
>     * Start an install
> 
>     * Wait till it is stuck
> 
>     * Click on any line in the `Console` tab then right-click it
> 
>     * Select `Save As` and send me the file here or on Discord to nistei#1362
> 
> 
> This way I can analyze and check what is happening.

@franklw262 be aware that posting on here might leak your folder structure that may contain sensitive information, depending on what you windows user name is.

**Comment by TLSY2** [comment]
maybe do it manual install from website.

**Comment by franklw262** [comment]
Hi and thanks for your help, unfortunately when the installer is in the locked mode I am unable to select any line from the console tab, so I have taken the above image, I hope it helps
Regards Frank

Sent from Mail for Windows 10
